xclim/core/checks.py

```diff
@@ -56,108 +79,15 @@ def assert_daily(var):
     # This won't work for non-standard calendars. Needs to be implemented in xarray. Comment for now
     if isinstance(t0.values, np.datetime64):
         if pd.infer_freq(var.time.to_pandas()) != "D":
-            raise ValueError("time series is not recognized as daily.")
+            raise ValidationError("time series is not recognized as daily.")
 
     # Check that the first time step is one day.
     if np.timedelta64(dt.timedelta(days=1)) != (t1 - t0).data:
-        raise ValueError("time series is not daily.")
+        raise ValidationError("time series is not daily.")
 
     # Check that the series does not go backward in time
     if not var.time.to_pandas().is_monotonic_increasing:
-        raise ValueError("time index is not monotonically increasing.")
-
-
-def check_valid_temperature(var, units):
-    r"""Check that variable is air temperature."""
-
-    check_valid(var, "standard_name", "air_temperature")
-    check_valid(var, "units", units)
-    assert_daily(var)
```

Comment:
Why remove this ?

Reply:
Hum. No solid reasons, before getting to this solution I began to write another that was not coherent with these "batch" checkers, which is why I removed them. But I can put them back, were they used elsewhere?  Sorry!

Reply:
I mean, this could also do with some refactoring and automating, possibly using cf-checker, but yes, let's keep them around until we replace them.